README.md

```diff
@@ -73,6 +73,25 @@ fn main() {
 }
 ```
 
+## Note about serde_derive Binary
+
+When using `serde_derive` either directly or via `serde` with `derive` feature.
+
+Please note that `serde_derive` may be distributed from within pre-compiled binary
```

Comment:
```suggestion
Please note that `serde_derive` is distributed with a pre-compiled binary used by default on x86_64 platforms.
```

Reply:
that is not entirely true though this binary only exists on x86_64-unknown-linux-gnu . Not other targets. Maybe something like this be better `Please not that serde_drive may be distributed from within pre-compiled binary (currently on x86_64-unknown-linux-gnu)` and this might turn to a table if more targets start to distribute with the pre-compiled binary.

Reply:
Rust is expressive and you can read the code for that level of detail. This is implementation detail of the decision.